core/src/main/scala/org/apache/spark/shuffle/FileShuffleBlockResolver.scala

```diff
@@ -23,6 +23,8 @@ import java.util.concurrent.atomic.AtomicInteger
 
 import scala.collection.JavaConversions._
 
+import com.google.common.annotations.VisibleForTesting
```

Comment:
I think this is a new standard we are trying to use -- you'll see its already in use in a lot of other places.  I didn't decide on using this particular annotation, but if i had to hazard a guess, its probably just because it was already there ...